src/Header/RequestHandler.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_REQUESTHANDLER_H_
+#define GROUP4_HEADER_REQUESTHANDLER_H_
+
+#include "OperateData.h"
```

Comment:
Please sort and order your includes according to the guide:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done